guides/source/active_storage_overview.md

```diff
@@ -539,60 +539,76 @@ Analysis requires the `mini_magick` gem. Video analysis also requires the [FFmpe
+
+```erb
+<ul>
+  <% @message.files.each do |file| %>
+    <li>
+      <% if file.representable? %>
+        <%= image_tag file.representation(resize_to_limit: [100, 100]) %>
+      <% else %>
+        <%= link_to url_for(file) do %>
+          Download file
+        <% end %>
+      <% end %>
+    </li>
+  <% end %>
+</ul>
+```
+
+Internally, `representation` calls `variant` for images, and `preview` for previewable files. You can also call these methods directly.
+
+[representing]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Representable.html
+
+### Transforming Images
+
+Transforming images allows you to display the image at dimensions of your choosing. To enable variants, add the `image_processing` gem to your `Gemfile`:
 
 ```ruby
 gem 'image_processing'
 ```
 
-To create a variation of an image, call [`variant`][] on the attachment. You can pass any transformation to the method supported by the processor. The default processor for Active Storage is MiniMagick, but you can also use [Vips](https://www.rubydoc.info/gems/ruby-vips/Vips/Image).
-
-When the browser hits the variant URL, Active Storage will lazily transform the
-original blob into the specified format and redirect to its new service
-location.
+To create a variation of an image, call [`variant`][] on the attachment. You can pass any transformation to the method supported by the processor. When the browser hits the variant URL, Active Storage will lazily transform the original blob into the specified format and redirect to its new service location.
 
 ```erb
 <%= image_tag user.avatar.variant(resize_to_limit: [100, 100]) %>
 ```
 
-To switch to the Vips processor, you would add the following to
-`config/application.rb`:
+If you don't want to transform images and just want the file that was uploaded, [link to the file](#linking-to-files).
+
+The default processor for Active Storage is MiniMagick, but you can also use [Vips][]. To switch to Vips, add the following to `config/application.rb`:
 
 ```ruby
-# Use Vips for processing variants.
 config.active_storage.variant_processor = :vips
 ```
 
 [`variant`]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Representable.html#method-i-variant
+[Vips]: https://www.rubydoc.info/gems/ruby-vips/Vips/Image
 
-Previewing Files
-----------------
+### Previewing Files
 
 Some non-image files can be previewed: that is, they can be presented as images.
 For example, a video file can be previewed by extracting its first frame. Out of
 the box, Active Storage supports previewing videos and PDF documents. To create
 a link to a lazily-generated preview, use the attachment's [`preview`][] method:
 
 ```erb
-<ul>
-  <% @message.files.each do |file| %>
-    <li>
-      <%= image_tag file.preview(resize_to_limit: [100, 100]) %>
-    </li>
-  <% end %>
-</ul>
+<%= image_tag message.video.preview(resize_to_limit: [100, 100]) %>
 ```
 
+To add support for another format, add your own [previewer][].
```

Comment:
See my comment about the `[representing][]` link.  You could link <code>[\`ActiveStorage::Previewer\`][]</code> instead.  Or we could just save this part for a future PR that includes more details.